woocommerce/woocommerce-ios
# Refunds Step 2 - Networking layer, Mark I

Partial work for #1307.

This PR adds the condensed version of refunds to an Order model in Networking. Unit testing via Circle CI will fail. Only local testing on Xcode will pass. I'm stopping here because this PR is already too big.

- removed previously written Networking code 
- added real JSON response for a fully refunded order
- added real JSON response for a partially refunded order
- added OrderRefundCondensed model to Networking and to Order model
- added OrderMapper unit tests
- added a Refunds feature flag

### To test
1. Target the Networking subproject
2. Run all unit tests for subproject

All Networking unit tests should pass.

Update release notes:
